README.md

```diff
@@ -105,3 +105,8 @@ As a minimum, any modules recommended as part of this reference architecture sho
 ## Contributing
 
 To Contribute to this project, please see the [Contributing Guide](./CONTRIBUTING.md).
+
+## Contributors
```

Comment:
That might be hard to keep up to date.

Reply:
@wtrocki I don't think we should remove people, as past contributions are still important.  What I'd like to have up front in the README.md  is the breadth of the contributors and different groups/scopes that they work on which have contributed to the end result.

Reply:
Agreed

Reply:
@wtrocki I see you approved, but thinking you should add yourself to the list as well :)

Reply:
could use this https://github.com/marketplace/actions/contribute-list